packages/contracts-bedrock/src/L2/GasPriceOracle.sol

```diff
@@ -93,7 +94,101 @@ contract GasPriceOracle is ISemver {
                 total += 16;
             }
         }
-        uint256 unsigned = total + overhead();
-        return unsigned + (68 * 16);
+        return total + overhead();
```

Comment:
I think you still want the (68*16) here as it accounts for the signature which isn't part of the unsigned tx that typically is what is sent to this function.

Reply:
oh i see! So folks are expected to pass an unsigned tx to this method, but the `DataGas` func in `op-geth` is post-signature, which is why they removed the `68` overhead in regolith?
https://github.com/ethereum-optimism/op-geth/blob/c50337a60a1309a0f1dca3bf33ed1bb38c46cdd7/core/types/rollup_l1_cost.go#L32-L36